hs_core/migrations/0067_auto_20221107_1513.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2022-11-07 15:13
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import hs_core.hs_rdf
+from hs_core.models import Relation, BaseResource, GeospatialRelation
+
+
+def migrate_relations_to_geoconnex(apps, schema_editor):
+    for relation in Relation.objects.filter(type="relation"):
+        res = BaseResource.objects.get(object_id=relation.object_id)
```

Comment:
You only need to retrieve the resource object when in the if block below.

Reply:
good point, thanks